instrument_registry/api/routes/health.py

```diff
@@ -0,0 +1,9 @@
+from fastapi import APIRouter
+from fastapi.responses import JSONResponse
+
+router = APIRouter()
+
+
+@router.get("/live", response_class=JSONResponse)
+async def liveness() -> dict[str, str]:
+    return {"status": "ok"}
```

Comment:
optionally: you could consider making the response a model or add typehints so the openapi specs can help the user better:
```
@router.get("/live", response_model=xxx, response_class=JSONResponse)
```

or you can even add more to the get
```
    summary="Liveness Check",
    description="This endpoint returns the liveness status of the application.",
    response_model=LivenessResponseModel,
    responses={
        200: {"description": "The application is live and running"}
    }
   ```

Reply:
you could consider doing the same for the other endpoints helping the users with examples and descriptions